README.md

```diff
@@ -52,13 +52,17 @@ The `.github/workflows/build-test.yml` builds a fully statically linked version
 
 ## Testing locally
 
+Create a ssh key called `test_key` at the root of the `hpci` repository.
```

Comment:
It might be worth adding an additional line (or a link to some docs) to explain how to generate a new SSH key at a non-standard location. E.g. https://superuser.com/questions/1004254/how-can-i-change-the-directory-that-ssh-keygen-outputs-to

Reply:
Addressed in https://github.com/Garvan-Data-Science-Platform/hpci/pull/6/commits/8dfe89e35b78fcbd9639b1cc34b483c8ee61c565

I tested and ssh-keygen will warn before overwriting an existing key FYI @ammazzaw